tests/test_sqs/test_sqs.py

```diff
@@ -229,20 +229,15 @@ def test_get_queue_url_errors():
 @mock_sqs
 def test_get_nonexistent_queue():
     sqs = boto3.resource("sqs", region_name="us-east-1")
-    with pytest.raises(ClientError) as err:
-        sqs.get_queue_by_name(QueueName="non-existing-queue")
-    ex = err.value
-    ex.operation_name.should.equal("GetQueueUrl")
-    ex.response["Error"]["Code"].should.equal("AWS.SimpleQueueService.NonExistentQueue")
-    ex.response["Error"]["Message"].should.equal(
-        "The specified queue non-existing-queue does not exist for this wsdl version."
-    )
 
     with pytest.raises(ClientError) as err:
         sqs.Queue("http://whatever-incorrect-queue-address").load()
```

Comment:
I started out with a more complete URL validator, until I saw this test. Running this against AWS itself results in this NonExistentQueue-error.

I'd expected it to return an InvalidAddress, because the URL does not adhere to the usual `http://amazonaws.com/queue/queue-name`, but I guess AWS doesn't do a full URL validation either